shared/js/background/helpers/arguments-object.js

```diff
@@ -26,10 +25,17 @@ export function getArgumentsObject (tabId, sender, documentUrl, sessionKey) {
         site.enabledFeatures = site.enabledFeatures.filter(feature => aboutBlankEnabled.includes(feature))
     }
 
+    site.enabledFeatures = site.enabledFeatures.filter((feature) => {
+        // Prune out the tracker allowlist feature setting as it's not needed and is large
+        if (feature === 'trackerAllowlist') return false
+
+        // Disable referrer trimming when we're not changing the referrer for the tab
+        if (feature === 'referrer' && !tab.referrer?.referrer) return false
```

Comment:
When the `tab.referrer` is blank we're expecting the feature to be disabled, this happened previously inside C-S-S prior to https://github.com/duckduckgo/content-scope-scripts/commit/5492c11b5c71cb5b1f1db5a131424a51f3f977f1

This was an actual bug once we've merged that code in C-S-S

Reply:
Do we still need https://github.com/duckduckgo/content-scope-scripts/commit/acc38de473f162e406d51c70ce0dcac75f58bbb6 then?

Reply:
Yeah I think so, the browser might choose to sever the referrer itself or other extensions. It's pretty unrelated to this issue.